CMakeLists.txt

```diff
@@ -47,7 +55,10 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/TorchVisionConfig.cmake
   DESTINATION ${TORCHVISION_CMAKECONFIG_INSTALL_DIR})
 
 install(TARGETS ${PROJECT_NAME}
-  EXPORT TorchVisionTargets)
+  EXPORT TorchVisionTargets
+  LIBRARY DESTINATION ${TORCHVISION_INSTALL_LIBRARY_DIR}
+  ARCHIVE DESTINATION ${TORCHVISION_INSTALL_ARCHIVE_DIR}
```

Comment:
Could you use `CMAKE_INSTALL_LIBDIR` as defined [here](https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html)?

Reply:
Thank you for your help. I've defined it.(I am not familiar with CMake.)